#! /bin/sh /usr/share/dpatch/dpatch-run
## fix-firefox-libraries.dpatch by Jose Carlos Garcia Sogo <jsogo@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fixes warning about Firefox libraries not found.
## DP: It ahs to search for xulrunner instead of firefox.

@DPATCH@
diff -urNad conduit.git~/conduit/conduit conduit.git/conduit/conduit
--- conduit.git~/conduit/conduit	2009-04-18 00:07:23.000000000 +0200
+++ conduit.git/conduit/conduit	2010-01-29 00:12:11.066656821 +0100
@@ -9,7 +9,7 @@
 # to run a program using mozilla libs
 
 if [ x$CONDUIT_BROWSER_IMPL = x"gtkmozembed" ]; then
-    for d in `ls -d /usr/lib*/firefox* /usr/lib*/mozilla-firefox* 2> /dev/null | sort -r`
+    for d in `ls -d /usr/lib*/firefox* /usr/lib*/mozilla-firefox* /usr/lib*/xulrunner* 2> /dev/null | sort -r`
     do
         if [ -x $d/run-mozilla.sh ]; then
             echo INFO: FOUND FIREFOX LIBS AT $d
